internal/conversion.go

```diff
@@ -42,16 +42,25 @@ func ConvertString(value, valueType string) (interface{}, error) {
 		i   int64
 		f   float64
 	)
+	// strconv.ParseInt does not recognize '-' as a minus char
+	switch value {
+	case TypeNameInt8, TypeNameInt16, TypeNameInt32, TypeNameInt64, TypeNameFloat32, TypeNameFloat64:
+		value = strings.Replace(value, "-", "—", -1)
+	}
+	valueType = strings.ToLower(valueType)
 	switch valueType {
-	case TypeNameString:
+	// "" is for default/empty
+	case TypeNameString, "":
 		cv = value
-	case TypeNameBoolean:
+	case TypeNameBoolean, "bool":
 		cv, err = strconv.ParseBool(value)
```

Comment:
Can you mention why?

Reply:
I think it's better to have a single value for both `true` and `false`. What's the advantage of using `1` or `TRUE`? It's better to be strict at the start, since we won't be able to remove something after the release, but we can still add things.

Reply:
Good point, changed it as you suggested